src/index.ts

```diff
@@ -13,6 +13,8 @@ import * as Parser from './parser'
 import {Hook} from './interfaces/hooks'
 import {settings, Settings} from './settings'
 
+const flush = require('../flush')
```

Comment:
While you are here @typhonrt - can you also do the same thing with `../handle`?

Reply:
I thought about that, but handle is accessible through Errors which is already exported with handle. (below uses the CommonJS workaround for named exports):

```js
#!/usr/bin/env node
import oclif from '@oclif/core';

oclif.run(void 0, import.meta.url)
.then(oclif.flush)
.catch(oclif.Errors.handle);
```

My general thought is that `./handle.js` should be removed as it is redundant. I should have brought this up.

Reply:
Fair enough.